integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1458,6 +1470,146 @@ public static void createDomainAndVerify(Domain domain,
     return ingressHostList;
   }
 
+  /**
+   * add security context constraints to the service account of db namespace.
+   * @param serviceAccount - service account to add to scc
+   * @param namespace - namespace to which the service account belongs
+   */
+  public static void addSccToDBSvcAccount(String serviceAccount, String namespace) {
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command("oc adm policy add-scc-to-user anyuid -z " + serviceAccount + " -n " + namespace))
+        .execute(), "oc expose service failed");
+  }
+
+  /**
+   * We need to expose the service as a route for ingress.
+   *
+   * @param serviceName - Name of the route and service
+   * @param namespace - Namespace where the route is exposed
+   */
+  public static String createRouteForOKD(String serviceName, String namespace) {
+    //String hostName = serviceName + "-" + namespace;
+    String hostName = serviceName;
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command("oc expose service " + serviceName + " -n " + namespace
+                + " --hostname=" + hostName))
+        .execute(), "oc expose service failed");
+    /*
+    String routeCommand = "oc expose service -n " + namespace;
+    logger.info("Exposing the service as a route using {0}", routeCommand);
+    ExecResult result = new Command()
+        .withParams(new CommandParams().command(routeCommand))
+        .executeAndReturnResult();
+    logger.info("Route exists? - {0}", result.toString());
+    */
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command("sudo sed -i \"/192.168.130.11/s/$/ ${serviceName}/\" /etc/hosts"))
+        .execute(), "added service name " + serviceName + " to etc hosts file");
+    // In OKD, it takes a little while for the route to be fully functional
+    try {
+      Thread.sleep(10 * 1000);
+    } catch (InterruptedException iex) {
+      //ignore
+    }
+    return hostName;
+  }
+
+  /**
+   * In OKD environment, the nodePort cannot be accessed directly. We need to create an ingress
+   *
+   * @param podName name of the pod - to create the ingress for its external service
+   * @param namespace namespace of the domain
+   * @return hostname to access the ingress
+   */
+  public static String createIngressForOKD(String podName, String namespace) {
```

Comment:
Is it ingress for Admin Server ONLY.  If so rename the method to something like createAdminIngressInOKD()